src/components/SharedComponents/ActivityCount.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import INatIcon from "components/INatIcon";
+import Body3 from "components/SharedComponents/Typography/Body3";
+import { View } from "components/styledComponents";
+import type { Node } from "react";
+import React from "react";
+import { t } from "i18next";
+
+type Props = {
+  numOfComments: number,
+  color: string,
+  icon?: string
+}
+
+const ActivityCount = ( { numOfComments, color, icon }: Props ): Node => (
+  <View
+    className="flex-row items-center"
+    accessible
+    accessibilityLabel={t( "Number-of-comments" )}
+    accessibilityValue={{ text: numOfComments.toString() }}
+  >
+    <INatIcon
+      name={icon || "comments-filled-in"}
+      color={color}
+      size={14}
+    />
+    <Body3
+      className="ml-1.5"
+      testID="ActivityCount.commentCount"
+      style={{ color }}
+    >
+      {numOfComments}
```

Comment:
We're going to need to localize numbers for display like this eventually, so let's feature creep this issue and add it here! The official i18next guidance is at https://www.i18next.com/translation-function/formatting. Even in English it should add commas to a number like 999999.